### ISSUE TITLE
Unsolvable error occurred for the second time after reinstalling win10 system

### ISSUE THREAD

**Issue by chengkeng** [NVlabs/stylegan2-ada-pytorch#130]
OS: Windows 10
PyTorch version: 1.7.1
CUDA toolkit version: 11.1
NVIDIA driver version: 460.89GPU RTX 3090
Docker: —

I used the pytorch version of stylegan2-ada in the first two months, everything was normal, but after I interrupted the task once, it suddenly became unavailable after this problem occurred. I think it may be that the pytorch version is not perfect, so I switched back to the tensorflow version of Stylegan2-ada.

Recently I reinstalled the operating system to use CUDA 11.1, and I used the pytorch version of stylegan2-ada again. Everything was normal when I started using it, but after I interrupted the task, I encountered the following problems again. 

I think it may still be the original tensorflow The version is more stable.

Questions are as follows:

python train.py --metrics=none  --outdir=./MyTrain --data=./datasets/mydataset.zip --aug=noaug --gamma=10 --cfg=auto --batch=60 --snap=5 --resume=ffhq256

Training options:
{
  "num_gpus": 1,
  "image_snapshot_ticks": 5,
  "network_snapshot_ticks": 5,
  "metrics": [],
  "random_seed": 0,
  "training_set_kwargs": {
    "class_name": "training.dataset.ImageFolderDataset",
    "path": "./datasets/mydataset.zip",
    "use_labels": false,
    "max_size": 1318,
    "xflip": false,
    "resolution": 256
  },
  "data_loader_kwargs": {
    "pin_memory": true,
    "num_workers": 3,
    "prefetch_factor": 2
  },
  "G_kwargs": {
    "class_name": "training.networks.Generator",
    "z_dim": 512,
    "w_dim": 512,
    "mapping_kwargs": {
      "num_layers": 8
    },
    "synthesis_kwargs": {
      "channel_base": 16384,
      "channel_max": 512,
      "num_fp16_res": 4,
      "conv_clamp": 256
    }
  },
  "D_kwargs": {
    "class_name": "training.networks.Discriminator",
    "block_kwargs": {},
    "mapping_kwargs": {},
    "epilogue_kwargs": {
      "mbstd_group_size": 4
    },
    "channel_base": 16384,
    "channel_max": 512,
    "num_fp16_res": 4,
    "conv_clamp": 256
  },
  "G_opt_kwargs": {
    "class_name": "torch.optim.Adam",
    "lr": 0.0025,
    "betas": [
      0,
      0.99
    ],
    "eps": 1e-08
  },
  "D_opt_kwargs": {
    "class_name": "torch.optim.Adam",
    "lr": 0.0025,
    "betas": [
      0,
      0.99
    ],
    "eps": 1e-08
  },
  "loss_kwargs": {
    "class_name": "training.loss.StyleGAN2Loss",
    "r1_gamma": 10.0
  },
  "total_kimg": 25000,
  "batch_size": 60,
  "batch_gpu": 60,
  "ema_kimg": 5.0,
  "ema_rampup": null,
  "resume_pkl": "https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/transfer-learning-source-nets/ffhq-res256-mirror-paper256-noaug.pkl",
  "ada_kimg": 100,
  "run_dir": "./MyTrain\\00024-mydataset-auto1-gamma10-batch60-noaug-resumeffhq256"
}

Output directory:   ./MyTrain\00024-mydataset-auto1-gamma10-batch60-noaug-resumeffhq256
Training data:      ./datasets/mydataset.zip
Training duration:  25000 kimg
Number of GPUs:     1
Number of images:   1318
Image resolution:   256
Conditional model:  False
Dataset x-flips:    False

Creating output directory...
Launching processes...
Loading training set...

Num images:  1318
Image shape: [3, 256, 256]
Label shape: [0]

Constructing networks...
Resuming from "https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/transfer-learning-source-nets/ffhq-res256-mirror-paper256-noaug.pkl"
Setting up PyTorch plugin "bias_act_plugin"...
